packages/block-editor/src/components/list-view/use-list-view-drop-zone.js

```diff
@@ -52,19 +58,91 @@ import { store as blockEditorStore } from '../../store';
  *                                                  'inside' refers to nesting as an inner block.
  */
 
+export const NESTING_LEVEL_INDENTATION = 28;
```

Comment:
Good idea to add comments to link these together (added in https://github.com/WordPress/gutenberg/pull/49742/commits/5e9068ce86d244dd23a9ca966b57d6c1c72fb7b0). I also really like the idea of switching the indentation over to JS styles so that the hard-coded indentation level is used in both places 👍

If / when this PR lands, I'm happy to dig into that as a follow-up!

Reply:
> I don't think it would be inappropriate at all to set marign-left on the expander using an inline style, rewrite this logic using JavaScript, and then that way the same code that's used for setting the margin-left can be used here for calculating the drop destination.

I wrote it in a similar way originally but using spacer divs (which would possibly have made this PR easier), it was changed here - https://github.com/WordPress/gutenberg/pull/32063. 

Another option could be injecting css vars for the level and indent, and then doing a `calc` to multiply them.

`list-view/index.js` also has `BLOCK_LIST_ITEM_HEIGHT` that could be improved in a similar way.

Sounds good for a follow-up.

Reply:
Good ideas, I'll dig into this stuff in follow-ups. Thanks!